bank-balance-exactly-once/src/main/java/com/github/simplesteph/udemy/kafka/streams/BankBalanceExactlyOnceApp.java

```diff
@@ -53,22 +47,21 @@ public static void main(String[] args) {
         initialBalance.put("time", Instant.ofEpochMilli(0L).toString());
 
         KTable<String, JsonNode> bankBalance = bankTransactions
-                .groupByKey(Serdes.String(), jsonSerde)
+                .groupByKey(Serialized.with(Serdes.serdeFrom(String.class),Serdes.serdeFrom(JsonNode.class)))
                 .aggregate(
                         () -> initialBalance,
                         (key, transaction, balance) -> newBalance(transaction, balance),
-                        jsonSerde,
-                        "bank-balance-agg"
+                        Materialized.as("bank-balance-agg")
```

Comment:
keep the `jsonSerde`, this line is good though

Reply:
All the versions of aggregate that will take jsonSerde, are deprecated.
[KGroupedStream](https://kafka.apache.org/10/javadoc/org/apache/kafka/streams/kstream/KGroupedStream.html)

Reply:
As same comment as above something like this should work:
```
Materialized.<String, JsonNode, KeyValueStore<Bytes, byte[]>>as("bank-balance-agg")
                                    .withKeySerde(Serdes.String())
                                    .withValueSerde(jsonSerde)
```